### PR TITLE
Docs: Update required access control permissions for data source permission api

### PR DESCRIPTION
**Pull request by kalleep**
**What this PR does / why we need it**:
Update docs to reflect correct permissions for the data source permission API

### CODE REVIEW

File: docs/sources/enterprise/access-control/permissions.md [hunk 1]
```diff
@@ -23,77 +23,75 @@ scope
...
+| `datasources:write`             | `datasources:*`<br>`datasources:id:*`                                                       | Update data sources.                                                                                                                                       |
+| `datasources:delete`            | `datasources:id:*`<br>`datasources:uid:*`<br>`datasources:name:*`                           | Delete data sources.                                                                                                                                       |
+| `datasources.permissions:read`  | `datasources:*`<br>`datasources:id:*`                                                       | List data source permissions.                                                                                                                              |
+| `datasources.permissions:write` | `datasources:*`<br>`datasources:id:*`                                                       | Update data source permissions.                                                                                                                            |
```
Comment (kalleep) [inline docs/sources/enterprise/access-control/permissions.md:89]: Only changed these lines

**Review by vtorosyan — APPROVED** [review]
Since this is just renaming and removing existing permissions, I think we can go ahead with a merge.

**Review by achatterjee-grafana — APPROVED** [review]

**Comment by achatterjee-grafana** [comment]
Needs a backport label to show docs in `'latest`.

**Comment by kalleep** [comment]
This changes will be in 8.4, do i still need to have a backport label @achatterjee-grafana?

**Comment by achatterjee-grafana** [comment]
> This changes will be in 8.4, do i still need to have a backport label @achatterjee-grafana?

Hi @kalleep, if the work was done for the 8.4 release, then we do not need to backport. Thanks!

**Pull request merged**